sklearn/mixture/tests/test_gmm.py

```diff
@@ -111,7 +121,11 @@ def test_lmvnpdf_full():
     reference = _naive_lmvnpdf_diag(X, mu, cv)
     lpr = mixture.log_multivariate_normal_density(X, mu, fullcv, 'full')
```

Comment:
you can do both at once was my point, the assert_warns_message returns the result of the function. And the way you currently do it will not suppress the deprecation warning.